administrator/components/com_media/src/Event/FetchMediaFileEvent.php

```diff
@@ -11,6 +11,7 @@
 
 \defined('_JEXEC') or die;
 
+use BadMethodCallException;
```

Comment:
Can you not remove this and use the global class inside with a beginning \

Reply:
yea, as sec

Reply:
done